pkg/innerring/processors/alphabet/handlers.go

```diff
@@ -1,13 +1,13 @@
 package alphabet
 
 import (
-	"github.com/nspcc-dev/neofs-node/pkg/innerring/timers"
 	"github.com/nspcc-dev/neofs-node/pkg/morph/event"
+	netmapEvent "github.com/nspcc-dev/neofs-node/pkg/morph/event/netmap"
 	"go.uber.org/zap"
 )
 
 func (ap *Processor) HandleGasEmission(ev event.Event) {
-	_ = ev.(timers.NewAlphabetEmitTick)
+	_ = ev.(netmapEvent.NewEpoch)
 	ap.log.Info("tick", zap.String("type", "alphabet gas emit"))
```

Comment:
not a `tick` anymore, lets actualize too

Reply:
fixed, added epoch to the logs